rest-api/dao/participant_summary_dao.py

```diff
@@ -345,6 +345,11 @@ def to_client_json(self, model):
          model.withdrawalTime < clock.CLOCK.now() - WITHDRAWN_PARTICIPANT_VISIBILITY_TIME)):
       result = {k: result.get(k) for k in WITHDRAWN_PARTICIPANT_FIELDS}
 
+    elif model.withdrawalStatus != WithdrawalStatus.NO_USE and \
+      model.suspensionStatus == SuspensionStatus.NO_CONTACT:
+      for i in SUSPENDED_PARTICIPANT_FIELDS:
+        result[i] = UNSET
```

Comment:
Just confirming that as soon as someone is suspended, these will get cleared out (unlike withdrawn participants, that will show up with everything for 48 hours.) So there isn't going to be an opportunity for folks at sites to reach out to them using their contact info after they suspend their account. (That may be totally fine... let's just make sure that's the intention.)

Reply:
Where does it say that? That's not how things are implemented now... suspended currently just means "don't contact me", not "don't accept data for this person" (unlike withdrawal)